debian/eos-payg-data.install

```diff
@@ -1,2 +1,3 @@
 usr/share/locale
 usr/bin/eos-payg-ctl
+usr/bin/get-clock-boottime
```

Comment:
Nitpick: Helper executables like this generally go in a subdirectory of `lib` or `libexec`, because they don’t need to appear in the `PATH`. i.e. `/usr/lib/eos-payg1/eos-payg-get-clock-boottime`. Maybe worth fixing if you have a moment, but not critical.